docs/update-strategy/update-strategy.adoc

```diff
@@ -613,4 +644,102 @@ with_retries istiod_pods_count "1"
 echo "Confirm istiorevision is deleted"
 with_retries istio_revisions_ready_count "1"
 print_istio_info
-endif::[]
\ No newline at end of file
+endif::[]
+
+[[updating-ambient-components]]
+== Updating Ambient Mode Components
+
+This section covers updating IstioCNI, ZTunnel, and Waypoint proxies after control plane updates. These procedures apply to both InPlace and RevisionBased update strategies.
```

Comment:
It's a bit confusing that we say earlier that RevisionBased update strategy is not possible to ambient mode and here we say it applies to both modes.